# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/cookie-store/__dir__.ini
# Commit: 2fc9a3d0ad8b
# Full Hash: 2fc9a3d0ad8bdbce325df78fcd731e925245afaf
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 1 - CookieStore API - WebIDL, r=smaug,edgul
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215144
# ==============================================================================

diff -r 2d598be5ec67 -r 2fc9a3d0ad8b testing/web-platform/meta/cookie-store/__dir__.ini
--- a/testing/web-platform/meta/cookie-store/__dir__.ini	Wed Sep 11 16:09:37 2024 +0000
+++ b/testing/web-platform/meta/cookie-store/__dir__.ini	Wed Sep 11 16:26:54 2024 +0000
@@ -1,3 +1,4 @@
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1475599
+prefs: [dom.cookieStore.enabled:true]
 implementation-status: backlog
 lsan-allowed: [Alloc, Malloc, Then, mozilla::BasePrincipal::CreateContentPrincipal, mozilla::dom::DocGroup::Create, mozilla::dom::ServiceWorkerJobQueue::RunJob, mozilla::dom::ServiceWorkerManager::Unregister, mozilla::dom::ServiceWorkerRegistrationMainThread::Unregister, mozilla::dom::UnregisterCallback::UnregisterCallback, mozilla::net::nsStandardURL::TemplatedMutator, operator]